python/dgl/runtime/scheduler.py

```diff
@@ -484,8 +484,9 @@ def schedule_pull(graph,
             schedule_apply_nodes(pull_nodes, apply_func, graph.dstframe, inplace,
                                  outframe, ntype=graph.canonical_etype[-1])
     else:
+        # TODO(Allen): Change operation to dgl operation
```

Comment:
Which operation?

Reply:
I was just thinking of changing all operation on index to dgl operation, without backend operation, to avoid potential unsupported or dtype casting issue. However I'm fine with the current code.